cf-custom-resources/lib/backlog-per-task-calculator.js

```diff
@@ -0,0 +1,123 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+"use strict";
+
+const aws = require("aws-sdk");
+
+// AWS Clients that are overriden in tests.
+let ecs, sqs;
+
+/**
+ * This lambda function calculates the backlog of SQS messages per running ECS tasks,
+ * and writes the metric to CloudWatch.
+ */
+exports.handler = async (event, context) => {
+  setupClients();
+  try {
+    const runningCount = await getRunningTaskCount(process.env.CLUSTER_NAME, process.env.SERVICE_NAME);
+    const timestamp = Date.now();
```

Comment:
Just a nit but maybe we should do it inside of `logBacklogPerTaskMetric` to make sure we are using the time right now.

Reply:
I wanted to share the same timestamp across all queues but it's a good point 💭 
Ideally it would happen after retrieving the backlogPerTask metric. I think we can refactor to do that

Reply:
Done ! https://github.com/aws/copilot-cli/pull/2804/commits/9ab295833960dfb732e9ccd7e3bd9e7f88646d4d